sharding/database/inmemory.go

```diff
@@ -54,3 +55,13 @@ func (sb *ShardKV) Delete(k []byte) error {
 	delete(sb.kv, common.BytesToHash(k))
 	return nil
 }
+
+func (sb *ShardKV) Close() {
+	//TODO: Implement Close for ShardKV
+	panic("ShardKV Close() isnt implemented yet")
```

Comment:
This can be a no-op and we can call `Close()` on `Stop()` for `ShardEthereum` or whatever constructs this library.

Reply:
This is in-memory key/value store, I think it's only used for tests